.circleci/config.yml

```diff
@@ -114,7 +114,24 @@ jobs:
       - run:
           name: Deploy
           command: npx hardhat deploy --network hardhat
-  
+
+  e2e-test:
+    working_directory:  ~/squeeth/packages/hardhat
+    docker:
+      - image: circleci/node:16.6.0
+    steps:
+      - restore_cache:
+          key: squeeth-{{ .Environment.CIRCLE_SHA1 }}
+      - restore_cache:
+        key: typechain-cache-{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Set Env Variables
+          command: echo "export INFURA_KEY=${INFURA_KEY}" >> $BASH_ENV
+      - run:
+          name: Mainnet E2E Test 
+          command: |
+            ./ci/e2e.sh	
+
   coverage:
```

Comment:
Added mainnet fork to CircleCI, still need to: 
- include it in jobs
- add INFURA_KEY in circleci vars

Reply:
cc @block902